scripts/src/javatests/org/oppia/android/scripts/maven/MavenDependenciesListWriterTest.kt

```diff
@@ -0,0 +1,338 @@
+package org.oppia.android.scripts.maven
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.oppia.android.scripts.proto.License
+import org.oppia.android.scripts.proto.MavenDependency
+import org.oppia.android.scripts.proto.MavenDependencyList
+import org.oppia.android.scripts.proto.PrimaryLinkType
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.PrintStream
+
+private const val DATA_BINDING_POM = "https://maven.google.com/androidx/databinding/" +
+  "databinding-adapters/3.4.2/databinding-adapters-3.4.2.pom"
+private const val PROTO_LITE_POM = "https://repo1.maven.org/maven2/com/google/protobuf/" +
+  "protobuf-lite/3.0.0/protobuf-lite-3.0.0.pom"
+private const val IO_FABRIC_POM = "https://maven.google.com/io/fabric/sdk/android/" +
+  "fabric/1.4.7/fabric-1.4.7.pom"
+private const val GLIDE_ANNOTATIONS_POM = "https://repo1.maven.org/maven2/com/github/" +
+  "bumptech/glide/annotations/4.11.0/annotations-4.11.0.pom"
+private const val FIREBASE_ANALYTICS_POM = "https://maven.google.com/com/google/firebase/" +
+  "firebase-analytics/17.5.0/firebase-analytics-17.5.0.pom"
```

Comment:
we can keep this inside the class as these are not getting used by any other class.

Reply:
Done.